.github/workflows/windows_debug.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           ctest \
           --output-on-failure \
             --build-config Debug \
-            --tests-regex tests.examples \
+            --tests-regex "tests.examples|cmake_build_dir_gtest_emulation_test" \
```

Comment:
This will have to move to the build stage (there is no test, only a build target). We could of course make it into a proper compilation test but that doesn't need to happen in this PR.

Reply:
Actually, this will do nothing with the gtest test disabled on Windows. It would be good to test the regular build tests (with hello world component) on Windows though.

Reply:
Sorry did that too quickly.. So I add `HPX_WITH_UNIT_TESTS=ON` to the configure step and add `cmake_build_dir_targets_test` (`hello_world_component` test) in the build step?

Reply:
Yes. The tests should already be enabled (by default) so you can just add the targets to the build step.

Reply:
thanks!